include/mq/api/Abilities.h

```diff
@@ -0,0 +1,200 @@
+/*
+ * MacroQuest: The extension platform for EverQuest
+ * Copyright (C) 2002-2023 MacroQuest Authors
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License, version 2, as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+
+#pragma once
+
+#include "eqlib/Globals.h"
```

Comment:
i don't think we actually need anything from eqlib in this header. instead we need string_view

Reply:
and mq/base/Common.h

Reply:
Ah, yeah, I had that because I started this as header only.  I can move the using eqlib into the cpp.